build.gradle

```diff
@@ -2,6 +2,7 @@ plugins {
     id 'application'
     id 'checkstyle'
     id 'de.fuerstenau.buildconfig' version '1.1.8'
+    id 'idea'
```

Comment:
Useful to prevent problems in IntelliJ with `BuildConfig`

Reply:
This would be a really good commit message that would help us remember/understand why this line was added.